apps/platform/src/subscriptions/SubscriptionController.ts

```diff
@@ -118,6 +119,7 @@ export const subscriptionCreateSchema: JSONSchemaType<SubscriptionParams> = {
 }
 
 router.post('/', async ctx => {
+    requireProjectRole(ctx, 'admin')
```

Comment:
I would roll with the middleware here. `router.post('/', projectRoleMiddleware('admin'), async ctx => {` should work. You can pass any middleware you want between the path and the handle. Brings it inline with how you are handling elsewhere and keeps the role stuff at a higher level

Reply:
oh duh forgot you could do that, changing for all similar cases